app/scripts/modules/core/src/cluster/filter/ClusterFilterService.ts

```diff
@@ -473,6 +473,12 @@ export class ClusterFilterService {
         if (oldGroup.entityTags || newGroup.entityTags) {
           oldGroup.entityTags = newGroup.entityTags;
         }
+        if (oldGroup.hasOwnProperty('isManaged') || newGroup.hasOwnProperty('isManaged')) {
```

Comment:
checking for only the `isManaged` field is sufficient, because the two fields travel together.